plugins/out_s3/s3.c

```diff
@@ -118,6 +118,44 @@ static char *mock_error_response(char *error_env_var)
     return NULL;
 }
 
+static void s3_retry_warn(struct flb_s3 *ctx, char *tag,
+                          char *input_name, time_t create_time,
+                          int less_than_limit)
+{
+    struct tm now_time;
+    char create_time_str[20];
+    struct tm *tmp;
+
+    tmp = localtime_r(&create_time, &now_time);
+    strftime(create_time_str, 20, "%Y-%m-%d %H:%M:%S", tmp);
+    if (input_name == NULL) {
+        if (less_than_limit == FLB_TRUE) {
+            flb_plg_warn(ctx->ins,
+                        "failed to flush chunk tag=%s, create_time=%s"
+                        "(out_id=%d)",
+                        tag, create_time_str, ctx->ins->id);
```

Comment:
So I know that in the original design, I said we want to as much as possible match the format of the normal retry messages... and we didn't want to add the "retry in X seconds" since we don't have a way of calculating the retry time... but note that I still had:

```
[ warn] [engine] failed to flush chunk tag=xxxxx, create_time=2022-08-18T21:34:42+0000, retry issued: input=forward.1 > output=s3.0 (out_id=0)
```

I think the `retry_issued` is important to let the user know clearly that we will retry